java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/JavaCheckVerifier.java

```diff
@@ -150,6 +151,17 @@ public static void verifyNoIssue(String filename, JavaFileScanner check) {
     scanFile(filename, check, javaCheckVerifier);
   }
 
+  public static void verifyNoIssueWithoutSemantic(String filename, JavaFileScanner check) {
+    JavaCheckVerifier javaCheckVerifier = new JavaCheckVerifier() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this a named "static" inner class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2694)